src/main/java/com/redhat/management/approval/Approver.java

```diff
@@ -77,20 +77,8 @@ public Approver(String userName, String emailAddress, String firstName, String l
     }
     
     public String toString() {
-        StringBuilder sb = new StringBuilder();
-        sb.append("Approver: ");
-        sb.append("\n firstName: ");
-        sb.append(this.firstName);
-        sb.append("\n lastName: ");
-        sb.append(this.lastName);
-        sb.append("\n userName: ");
-        sb.append( this.userName);
-        sb.append("\n emailAddress: ");
-        sb.append( this.emailAddress);
-        sb.append("\n randomAccessKey: ");
-        sb.append(this.randomAccessKey);
-
-        return sb.toString();
+        return String.format("Approver: \n firstName: %s\n lastName: %s\n userName: %s\n emailAddress: %s\n randomAccessKey: %s",
```

Comment:
`StringBuilder` is good in this method. It may be more readable in this way:
```
StringBuilder sb = new StringBuilder();
sb.append("Approver: ")
  .append("\n firstName: ").append(this.firstName)
  .append("\n lastName: ").append(this.lastName)
...
```